Formula/llvm.rb

```diff
@@ -130,6 +130,7 @@ class Llvm < Formula
   option "with-python", "Build bindings against custom Python"
   option "with-shared-libs", "Build shared instead of static libraries"
   option "without-libffi", "Do not use libffi to call external functions"
+  option "without-experimental", "Do not build libc++experimental library"
```

Comment:
> if it's experimental, I'd think we'd want it opt-in or not in the formula at all

Probably the latter given the higher bar for option justification these days. I was just curious as to whether this breaks any existing use case, and if it doesn't, whether it was worth chucking it in by default.